src/github/logic.py

```diff
@@ -248,6 +253,7 @@ def maybe_automerge_pull_request(pull_request: PullRequest) -> bool:
         not SGTM_FEATURE__AUTOMERGE_ENABLED
         or not _pull_request_is_open(pull_request)
         or pull_request.is_in_merge_queue()
+        or pull_request.base_ref_associated_pull_requests() > 0
     ):
         logger.info(
             f"Skipping automerge for {pull_request.id()} because it is closed or in merge queue"
```

Comment:
```suggestion
            f"Skipping automerge for {pull_request.id()} because it is closed, in merge queue, or base branch has associated PR(s)"
```